fqd.c

```diff
@@ -158,15 +164,71 @@ static uint32_t get_my_ip(void) {
   }
   return 0;
 }
+void signal_handler(int sig, siginfo_t *info, void *ucontext) {
+	bcd_fatal("This is a fatal crash");
+	raise(sig);
+	return;
+}
 int main(int argc, char **argv) {
+  struct sigaction new_action;
+  bcd_error_t error;
+	bcd_t bcd;
+  new_action.sa_handler = signal_handler;
```

Comment:
The signal_handler function needs to be updated to call bcd_fatal. See the README in bcd for how to call bcd_fatal.